lib/utils.h

```diff
@@ -78,4 +78,7 @@ inline static const char *getOrdinalText(int i)
     return "th";
 }
 
+template<typename T>
+inline void UNUSED(T&&) {}
```

Comment:
UNUSED could be implemented as (void)param, too (as a macro), it does not need to be a template.